### DISCUSSION TITLE
Simple Functionality Question

### DISCUSSION THREAD

**Discussion by 171stStreetGames** [julianxhokaxhiu/FFNx#709] [General]
As far as I'm aware, there is a way to toggle random encounters on and off. I can see the notes that say it's there. I have the most current version of FFNx. But I cannot for the life of me find out how to do it. I know the function exists, but I do not know where to go or what to do in either 7th Heaven or the game itself to actually utilize it.

**Comment by julianxhokaxhiu** [comment]
Hello, you can find all them documented here: https://github.com/julianxhokaxhiu/FFNx/blob/master/docs/shortcuts.md :) Enjoy
